qiita_db/analysis.py

```diff
@@ -266,6 +275,21 @@ def name(self):
         return conn_handler.execute_fetchone(sql, (self._id, ))[0]
 
     @property
+    def portal(self):
```

Comment:
Do we actually want to retrieve the portal as a property? In my opinion, the portal is something that happens in a transparent way in the system.